src/sage/numerical/interactive_simplex_method.py

```diff
@@ -2611,10 +2621,18 @@ def run_revised_simplex_method(self):
         self._final_revised_dictionary = d
         return HtmlFragment("\n".join(output))
 
-    def run_simplex_method(self):
+    def run_simplex_method(self, pivot_rule=None, *args):
```

Comment:
I would suggest to use `**kwds` instead of `*args` for passing parameters down to the pivot rule. This feels more idiomatic.

Perhaps even make `pivot_rule` a keyword-only parameter (so that one has to call it as `run_simplex_method(pivot_rule='steepest_edge')`). This can be done like this:
`def run_simplex_method(self, *, pivot_rule=None, **kwds):`